include/PreCharge/PreCharge.hpp

```diff
@@ -70,10 +71,22 @@ class PreCharge {
 
     /**
      * Constructor for pre-charge state machine
+     * 
+     * @param[key]: GPIO for motorcycle key
```

Comment:
```suggestion
     * @param[in] key GPIO for motorcycle key
```
This is what I meant by `param[in]` format